src/ServicePulse.Host/app/js/views/archive/controller.js

```diff
@@ -72,35 +73,53 @@
             vm.loadingData = false;
         };
 
+        var saveGroupOption = (amount, unit) => {
```

Comment:
Still not a fan of the name `default` here. Maybe `saveSelectedArchiveGroup` or something a bit more descriptive?

Reply:
The difference between this name and the one for classification below, is that in the classification one, if there isn't a cookie value a default value is used. In this case, if there is no cookie value then `undefined` (or `null`?) is used.

Reply:
yes...the `undefined` value will translate to 'All Archive' option in the filter.